tools.go

```diff
@@ -12,6 +12,7 @@ import (
 	_ "github.com/mna/pigeon"
 	_ "golang.org/x/tools/cmd/goimports"
 	_ "golang.org/x/tools/cmd/stringer"
+	_ "gopkg.in/yaml.v2"
```

Comment:
What is this used for?

Reply:
There was a test called `influxql_validation` that was failing:
https://app.circleci.com/pipelines/github/influxdata/influxdb/20730/workflows/b6570808-a0f1-478e-8f7f-2f95fd6b02ca/jobs/191747
Apparently this test needs this dependency, but it's not kept around by `go mod tidy`, and it can't be added as the test runs due to `-mod=readonly`. I added it to `tools.go` so that there is an explicit dependency. We've used this mechanism in the past in order to pin to versions of packages that are not necessarily used from Go code explicitly.

Is there a better way to address this?

Reply:
Nah that makes sense to me :+1: I was just curious